frontend/src/components/general/form/FormError.tsx

```diff
@@ -7,7 +7,7 @@ const FormError = ({ error, show }: FormErrorProps) => {
   return (
     <div className={`text-red font-semibold text-sm mt-10 ${show ? '' : 'hidden'}`}>
       <p className="mb-2">
-        Es ist leider ein Problem aufgetreten. Bitte probieren Sie es erneut oder wenden Sie sich an
+        Es ist leider ein Problem aufgetreten. Bitte probiere es erneut oder wenden dich an
```

Comment:
I think the term "versuchen" sounds better than "probieren"

```suggestion
        Es ist leider ein Problem aufgetreten. Bitte versuche es erneut oder wenden dich an
```